keps/prod-readiness/sig-node/2403.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2403
+stable:
```

Comment:
Thanks for your approval @johnbelamaric I think to make the verification lane happy this should read "alpha", right?

Reply:
reviewed other KEPs and changed to `alpha`.

Reply:
ok, the lane passed, it was that :)